docs/UserGuide.md

```diff
@@ -453,46 +449,75 @@ Examples:
 * If a `INTERVAL TIME_PERIOD` is provided for what was originally a non-recurring task, the edit will transform it into a recurring one based on the given frequency
+* Tasks are automatically sorted in chronological order upon modification, i.e. if a task on `25-10-22` is edited to be `30-10-22`, its new `TASK INDEX` would be based on the displayed order in the patient's task list.
+* If a `TIME` of `2400` if provided, it would simply be translated into `0000` hours of the next day. <br>
+  e.g `2-11-22 2400` will be automatically be inferred as `3-11-22 0000`.
+* If the day portion of the date exceeds the last day of that calendar month, it would default to the last day of the month. <br>
+  e.g `31-4-22` will be automatically converted to `30-4-22` or `30-2-20` will be converted to `29-2-20` since 2020 is a leap year.
+
+</div>
 
 Examples:
 * `list` followed by `edit -p 1 -d 1 d/Administer 3ml of example medicine` edits the description of the 1st task of the 1st patient in the patient list to `Administer 3ml of example medicine`, while retaining the original date and time for the task.
-* `find Betsy` followed by `edit -p 2 -d 3 d/| 23-10-22 0800` edits the date and time of the 3rd task of the 2nd patient in results of the `find` command to 23rd October 2022 0800 hours.
+* `find Betsy` followed by `edit -p 2 -d 3 d/| 23-10-22 0800` edits the date and time of the 3rd task of the 2nd patient in results of the `find` command to 23rd October 2022 0800 hours, while retaining the original description for the task.
+* `list` followed by `edit -p 1 -d 1 d/| | 3 days` edits the recurrence of the 1st task of the 1st patient if the task was a recurring task to every 3 days, while keeping the original description, date and time. If the task was a non-recurring task, then this edit transforms the task into a recurring task with a recurrence of every 3 days.
+* `list` followed by `edit -p 2 -d 3 d/| 25-10-22 | 2 weeks` edits the date and recurrence of the 3rd task fo the 2nd patient in the patient list to 25th october 2022 and every 2 weeks, while keeping the original description and time. If the task was a non-recurring task, then this edit transforms the task into a recurring task with a recurrence of every 2 weeks.
+* `find David` followed by `edit -p 1 -d 2 d/Change bandage | | 4 days` edits the description and recurrence of the 2nd task of the 1st patient in the results of `find` command to `Change bandage` and every 4 days, while keeping the original date and time. If the task was a non-recurring task, then this edit transforms the task into a recurring task with a recurrence of every 4 days.
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Notes:**
+* Note that you cannot edit only the time portion of the task, to change the time you would have to provide the date as well, e.g to change from `25-10-22 0800` to `25-10-22 0900`, the edit would have to involve the full date and time `d/ | 25-10-22 0900`.
```

Comment:
Several issues:
* Sentence is unnecessarily long. I'm sure there is a more concise way to phrase this.
* The spacing between `d/` and `|` is not consistent.
* The word `Note` at start of the sentence is redundant because you already have the box already tells the reader it is something to take note of.
* Move the example to a separate bullet point.

Reply:
rule of thumb: get straight to the point.
"You must always provide a `DATE` if you want to change the `TIME`". Since you're providing an example already, you don't have to worry about your initial explanation being too brief.